app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -53,30 +52,24 @@ const useStyles = makeStyles((theme) => ({
   buttonSecondaryText: {
     color: theme.palette.grey[600],
     fontWeight: 700,
+    fontSize: theme.typography.button.fontSize,
     transition: `color ${theme.transitions.duration.short} ${theme.transitions.easing.easeInOut}`,
```

Comment:
```suggestion
    transition: `color ${theme.transitions.duration.short}ms ${theme.transitions.easing.easeInOut}`,
```
sorry looking at the generated css looks like "ms" is necessary when you use the shorthand transition property.